libraries/libxml/source/org/pentaho/reporting/libraries/xmlns/parser/DomTreeResourceFactory.java

```diff
@@ -69,12 +70,13 @@ public Object getResource() {
   public Resource create( final ResourceManager manager,
                           final ResourceData data,
                           final ResourceKey context ) throws ResourceCreationException, ResourceLoadingException {
-
-    final DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
-    dbf.setNamespaceAware( true );
-    dbf.setValidating( false );
-
+    final DocumentBuilderFactory dbf;
     try {
+      dbf = DocumentBuilderFactory.newInstance();
```

Comment:
Suggest this part gets moved to libbase (the documentbuilderfactory.newInstance() and security stuff) so that we're moving that code all into a single place in the lib*.* world.